collections/map_not_nullish.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * Returns a new array, containing all elements in the given array transformed using the given transformer, except the ones
+ * that were transformed to `null` or `undefined`
+ *
+ * Example:
+ *
+ * ```typescript
```

Comment:
We prefer `ts` tag, but let's address this later.

Reply:
Won't be typechecked, but I'll address that upstream in cli.

Reply:
Oh sorry - I keep forgetting that. Should I post a separate PR?